### PR TITLE
Added Avatar component

### PR DESCRIPTION
**Pull request by Nagashree01**
Added avatar component having 3 different size: large, medium, small.
avatar

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import url('/stylesheets/base.css');
+.avatar{
+    border-radius: var(--border-rad-circle);
+    padding: 10px;
+}
```
Comment (mariyasada) [inline components/avatar.css:5]: fix the typo of rad

File: doc.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    overflow: auto;
+    padding-top: 10px;
+}
+
+
+
+
+
+
+
```
Comment (mariyasada) [inline doc.css:74]: remove unnecessary space

File: documentation.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                </div>
+
+                <div class="code-snippet">
+                    <iframe src="https://www.thiscodeworks.com/embed/6200af374dd44f0015f3cd32" style="width: 100%; height: 151px;" frameborder="0">
+                    </iframe>
+
+
+                </div>
+
+
```
Comment (mariyasada) [inline documentation.html:74]: remove unnecessary space

File: stylesheets/base.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    --text-lg:2.5rem;
+    --text-m:1.8rem;
+    --text-sm:1.2rem;
+
+
+
+
+
+
+
```
Comment (mariyasada) [inline stylesheets/base.css:37]: remove unnecessary space

File: stylesheets/navbar.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+.navigation a {
+    text-decoration: none;
+}
+
+.navigation-list {
+    display: flex;
+    justify-content: flex-end;
+    gap: 10px;
+}
```
Comment (mariyasada) [inline stylesheets/navbar.css:23]: use rem instead of px

**Review by mariyasada — APPROVED** [review]
look at the comments of mine

File: stylesheets/base.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+body {
+    font-family: 'Montserrat', sans-serif;
+}
+
+
+.flex{
+    display:flex;
+}
+
+.flex .column{
```
Comment (JantuDeb) [inline stylesheets/base.css:48]: You should remove `.flex` by default flex direction should be row.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: doc.css (hunk 2 above)
Comment (Nagashree01) [reply doc.css]: yes I'll definately do this

File: stylesheets/base.css (hunk 6 above)
Comment (Nagashree01) [reply stylesheets/base.css]: oh yes! I'll do this

**Pull request merged**